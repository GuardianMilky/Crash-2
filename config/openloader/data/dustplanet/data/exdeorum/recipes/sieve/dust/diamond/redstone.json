{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:dust"},"mesh":"exdeorum:diamond_mesh","result":"minecraft:redstone","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.12}}