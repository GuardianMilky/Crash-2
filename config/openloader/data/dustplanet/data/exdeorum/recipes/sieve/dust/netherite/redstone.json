{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:dust"},"mesh":"exdeorum:netherite_mesh","result":"minecraft:redstone","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.14}}