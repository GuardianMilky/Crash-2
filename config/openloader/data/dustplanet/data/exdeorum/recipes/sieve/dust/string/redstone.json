{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:dust"},"mesh":"exdeorum:string_mesh","result":"minecraft:redstone","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.06}}