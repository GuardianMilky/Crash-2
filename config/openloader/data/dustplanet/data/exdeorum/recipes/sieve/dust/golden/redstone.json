{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:dust"},"mesh":"exdeorum:golden_mesh","result":"minecraft:redstone","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.12}}